cli-plugins/insight/cli-configuration.hbs.md

```diff
@@ -7,26 +7,13 @@ This topic explains how to configure the Tanzu Insight plug-in.
 
 ## <a id='set-tar-cert'></a>Set the target and certificate authority certificate
 
-Set the target endpoint and CA certificate by running:
+Note: The following are instructions for ingress enabled, which is the recommended configuration. For instructions on non-ingress setups, see [Using Encryption and Connection](../../scst-store/using-encryption-and-connection.hbs.md) for more details.
```

Comment:
> The following are instructions for ingress enabled

This part feels a little awkward to me. What about something like: 
```
These instructions are for the recommended configuration where ingress is enabled. 
```

Reply:
While *Using Encryption and Connection* technically does contain links to LoadBalancer and NodePort. It's all the way at the bottom.

Can you link directly to the *Additional Resources* heading on that page to help the user?